resources/views/app.blade.php

```diff
@@ -30,10 +30,12 @@
 				<img src="/assets/img/laravel-auditing-logo.png" alt="Laravel Auditing logo" style="margin-left: 15px;margin-right: 0;height: 40px;top: 14px;">
 				Laravel Auditing
 			</a>
-			
-			<div class="responsive-sidebar-nav">
-				<a href="#" class="toggle-slide menu-link btn">&#9776;</a>
-			</div>
+
+			@if (!Request::is('/'))
```

Comment:
This will hide the button in any media, is that right?

Reply:
I think you should use hidden-xs

Reply:
Yes, any media.